db/migrate/20190716172246_create_company_settings.rb

```diff
@@ -2,13 +2,11 @@ class CreateCompanySettings < ActiveRecord::Migration
   def change
     create_table :company_settings do |t|
       t.integer :max_questions
-      t.boolean :is_sup_create_surv
-      t.boolean :is_sup_edit_surv
-      t.boolean :is_my_settings_sup
-      t.boolean :is_my_settings_memb
-      t.time :survey_expiry
-      t.integer :company_id
-      t.belongs_to :company, index: true
+      t.boolean :supervisors_survey_permission
+      t.boolean :supervisors_settings_permission
+      t.boolean :members_settings_permission
+      t.integer :survey_expiry
```

Comment:
is survey_expiry an integer?

Reply:
In company_settings, we decided it as integer as hours like 1, 2 etc.